### PR TITLE
Update DynamoDB support to the helm chart

### PR DESCRIPTION
**Pull request by scalar-boney**
DynamoDB support integration with the helm chart

### CODE REVIEW

File: charts/stable/schema-loading/templates/batchjob-schema-loading.yaml [hunk 1]
```diff
@@ -29,14 +27,22 @@ spec:
...
         {{- else if eq .Values.schemaLoading.database "cosmos" }}
         - "--cosmos"
+        - "-h"
+        - "{{ .Values.schemaLoading.contactPoints }}"
         - "-r"
         - "{{ .Values.schemaLoading.cosmosBaseResourceUnit }}"
+        {{- else if eq .Values.schemaLoading.database "dynamo"}}
+        - "--dynamo"
+        - "--region"
+        - "{{ .Values.schemaLoading.contactPoints }}"
```
Comment (feeblefakie) [inline charts/stable/schema-loading/templates/batchjob-schema-loading.yaml:45]: For not, it is fine, but I feel we should unify the schema tool's parameter for `contractPoints`.

**Review by feeblefakie — APPROVED** [review]
LGTM! Did you test?

**Comment by feeblefakie** [comment]
@mathewseby One of the CIs is failing. Can you check?

**Comment by scalar-boney** [comment]
> Did you test?

Yes

**Review by ymorimo — APPROVED** [review]
LGTM!

**Comment by feeblefakie** [comment]
@mathewseby I updated the doc with `scripts/update-chart-docs.sh`.

**Comment by mathewseby** [comment]
@feeblefakie san,
> @mathewseby I updated the doc with `scripts/update-chart-docs.sh`.

ooh, I was checking it. Thank you.

**Pull request merged**